tests/test_isr.py

```diff
@@ -141,7 +141,7 @@ def test_isr_basic(self):
         """Test basic ISR."""
         butler = lsst.daf.butler.Butler(self.repo, instrument='DECam', collections=self.basic_collection)
 
-        self.assertTrue(butler.datasetExists('postISRCCD', exposure=EXPOSURE, detector=DETECTOR))
+        self.assertTrue(butler.exists('postISRCCD', exposure=EXPOSURE, detector=DETECTOR))
```

Comment:
I'm actually inclined to delete this line completely. I don't think it gives us anything that we wouldn't learn from the next line which tries to get the dataset.

Reply:
👍 Agree.